modules/ROOT/pages/gov-cloud-standalone.adoc

```diff
@@ -0,0 +1,45 @@
+= Standalone Mule Support in MuleSoft Government Cloud
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MuleSoft Government Cloud deployment permits deployment of Mule runtime engines into an on-premises environment (standalone Mules). Standalone Mules, and
+MuleSoft support for standalone Mules, are outside the scope of FedRAMP. You are solely responsible for ensuring that all data shared with Salesforce complies
+with any and all data and information protection policies, requirements, and standards. Review the MuleSoft System Security Plan for FedRAMP scope.
+
+Rigorous internal security and compliance reviews were performed on services that support standalone Mule deployments in MuleSoft Government Cloud. 
+The Significant Change Request Form and updated System Security Plan were approved by our FedRAMP sponsoring agency.
+
+For FIPS compliance, Mule 4 agent version 2.4.9 or later is required.
+
+Standalone Mule support enables federal, state, and local government customers to deploy Mules inside their private cloud or 
+on-premises environment and manage them using the MuleSoft Government Cloud control plane. Key features include:
+
+* Mule Management
+
+** Unified view of applications, servers, and APIs
+** Application deployment and management across all environments
+** Integration with continuous deployment pipelines
+** xref:runtime-manager::index.adoc[Anypoint Runtime Manager] capabilities
+
+* Mule Monitoring
```

Comment:
Can we change the title of this section to: "Anypoint Runtime Manager Monitoring"